.gitignore

```diff
@@ -13,3 +13,5 @@ artifacts
 deployments
```

Comment:
this repo is missing gh actions. Can you add all actions from here?
https://github.com/axelarnetwork/axelar-gmp-sdk-solidity/tree/main/.github/workflows
Also, see solhint and solcover config files in root folder in gmp-sdk repo